python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1171,6 +1171,52 @@ def _get_abs_layer_name(node):
                                                      params, num_layers)
         return sym
 
+_loop_nodes = ['Merge', 'Switch', 'NextIteration', 'Exit', 'Enter', 'LoopCond']
+
+class Loop:
```

Comment:
this diff is not needed as it's already in by https://github.com/dmlc/tvm/commit/2df3364b0594d15053e23cee038305a6b52f144e#diff-eae8ecf976e0031823eeae454466f964R1278